### PR TITLE
Alena burak/6 3 coach check 4

### PR DESCRIPTION
**Pull request by ElenaBurak**

### CODE REVIEW

**Review by artemgael96 — APPROVED** [review]

File: exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public class PassedTestForCheckDTO
+    {
+        public int Id { get; set; }
+
+        public LevelType LevelType { get; set; }             
+
+        public StatusType Status { get; set; }
+
+       // public TestDTO UserTest { get; set; }
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs:14]: ?

File: exalearn-api/ExaLearn.Bl/DTO/PassedTestDTO.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using Shared.Enums;
+﻿using ExaLearn.Dal.Entities;
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/DTO/PassedTestDTO.cs:1]: Remove

**Review by vlad12345123 — APPROVED** [review]

File: exalearn-api/ExaLearn.Dal/Interfaces/IUserRepository.cs [hunk 3]
```diff
@@ -6,5 +6,7 @@ namespace ExaLearn.Dal.Interfaces
     public interface IUserRepository : IGenericRepository<User>
     {
         Task<string> GetUserRole(int id);
+
+        Task<int> FindByNameAsync(string userName);
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Interfaces/IUserRepository.cs:10]: Remove

File: exalearn-api/ExaLearn.Dal/Repositories/UserRepository.cs [hunk 4]
```diff
@@ -20,5 +20,12 @@ public async Task<string> GetUserRole(int id)
 
             return role.Name;
         }
+
+        public async Task<int> FindByNameAsync(string userName)
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Repositories/UserRepository.cs:24]: Remove

File: exalearn-api/ExaLearn.WebApi/appsettings.json [hunk 5]
```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=Pass12345;Database=exalern_db",
+    "DbContext": "Server=localhost;Password=123456;Port=5432;Username=postgres;Password=123;Database=exalern_db",
```
Comment (romanstat) [inline exalearn-api/ExaLearn.WebApi/appsettings.json]: Remove file from pr

**Pull request merged**